ons_ras_common/ons_rest_exercise.py

```diff
@@ -0,0 +1,38 @@
+##############################################################################
+#                                                                            #
+#   Generic Configuration tool for Micro-Service environment discovery       #
+#   License: MIT                                                             #
+#   Copyright (c) 2017 Crown Copyright (Office for National Statistics)      #
+#                                                                            #
+##############################################################################
+#
+#   ONSCase wraps routines used to access the case service
+#
+##############################################################################
+
+
+class ONSExercise(object):
+    """
+    This class is designed to take all the work out of accessing the case service. Initially it
```

Comment:
Why are there three classes which purport to take the work out of accessing the case service? And why is it so much work to access the case service?